### DISCUSSION TITLE
IPv6 DEFAULT ?

### DISCUSSION THREAD

**Discussion by wowik98** [wg-easy/wg-easy#236] [Q&A] [answered]
how to set IPv6 default to my docker-compose.yml please show me approximately how to install it

version: "3.8"
services:
  wg-easy:
    environment:
      # ⚠️ Required:
      # Change this to your host's public address
      - WG_HOST= 
      - WG_PORT=2929
      - WG_DEFAULT_DNS=84.200.69.80,84.200.70.40
      - PASSWORD= 
      - WG_DEFAULT_ADDRESS=10.66.66.x 
      - WG_ALLOWED_IPS=10.66.66.2/32, fddd:2c4:2c4:2c4::2/128
      - WG_PERSISTENT_KEEPALIVE=25
      # Optional:
      # - PASSWORD=foobar123
      # - WG_PORT=51820
      # - WG_DEFAULT_ADDRESS=10.8.0.x
      # - WG_DEFAULT_DNS=9.9.9.9
      # - WG_MTU=1420
      # - WG_ALLOWED_IPS=192.168.15.0/24, 10.0.1.0/24
      
    image: weejewel/wg-easy
    container_name: wg-easy
    volumes:
      - .:/etc/wireguard
    ports:
      - "2929:51820/udp"
      - "3030:51821/tcp"
    restart: unless-stopped
    cap_add:
      - NET_ADMIN
      - SYS_MODULE
    sysctls:
      - net.ipv4.ip_forward=1
      - net.ipv4.conf.all.src_valid_mark=1

**Comment by pheiduck** [comment, marked as answer]
Someone has heard your voice ;) See: https://github.com/wg-easy/wg-easy/pull/191
